app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInActivity.kt

```diff
@@ -167,7 +166,11 @@ class LoggedInActivity : BaseActivity(R.layout.activity_logged_in) {
         loggedInViewModel.data.observe(this) { data ->
             data?.let { d ->
                 if (binding.bottomNavigation.menu.isEmpty()) {
-                    val keyGearEnabled = isDebug() || d.member.features.contains(Feature.KEYGEAR)
+                    val keyGearEnabled: Boolean = if (shouldOverrideFeatureFlags(application as HedvigApplication)) {
```

Comment:
```suggestion
                    val keyGearEnabled = if (shouldOverrideFeatureFlags(application as HedvigApplication)) {
```

Also, should we do the same for `referralsEnabled`?